android-json-form-wizard/src/main/java/com/vijay/jsonwizard/NativeFormLibrary.java

```diff
@@ -12,6 +12,7 @@
 
     private ClientFormContract.Dao clientFormDao;
     private static NativeFormLibrary instance;
+    private static boolean performFormsTranslation = false;
```

Comment:
this shouldn't be static

Reply:
also refactor to singular `performFormTranslation`

Reply:
Done